addons-l10n/en/clones.json

```diff
@@ -1,4 +1,3 @@
 {
-  "clones/clones": "clones: {cloneCount}",
-  "clones/blank": "{cloneCount}"
```

Comment:
wl....this breaks the pr

Reply:
I'm aware, but it makes no sense to have a message without contents

Reply:
> I'm aware, but it makes no sense to have a message without contents

Sorry, actually I don't have the faintest notion what is that very JavaScript, so this was the best solution I could figure out without any notable experience.

Reply:
How about replacing `msg("blank", { cloneCount: i })` with simply just `i`?

Reply:
Wait wut?!? Processing that solution exceeds my brain capacity.

Reply:
> How about replacing `msg("blank", { cloneCount: i })` with simply just `i`?

Thanks a lot, it worked.